src/features/session_replay/aggregate/index.js

```diff
@@ -301,12 +305,14 @@ export class Aggregate extends AggregateBase {
         attributes: encodeObj({
           ...(this.shouldCompress && { content_encoding: 'gzip' }),
           'replay.firstTimestamp': firstTimestamp,
+          'replay.firstTimestampOffset': firstTimestamp - agentOffset,
           'replay.lastTimestamp': lastTimestamp,
           'replay.durationMs': lastTimestamp - firstTimestamp,
           'replay.nodes': this.events.length,
           'session.durationMs': agentRuntime.session.getDuration(),
           agentVersion: agentRuntime.version,
           session: agentRuntime.session.state.value,
+          rst: relativeNow,
           hasMeta: this.hasMeta,
           hasSnapshot: this.hasSnapshot,
           hasError: this.hasError,
```

Comment:
Late to ask but should this share the same or common base as the default query in harvest.js used by all the other feats?

Reply:
unfortunately no, since the base query is formatted differently than the blob endpoint requires. see `browser_monitoring_key` etc

Reply:
specifically this area
<img width="518" alt="Screenshot 2023-11-10 at 10 43 20 AM" src="https://github.com/newrelic/newrelic-browser-agent/assets/24296585/81f35bf3-0de0-4d1a-9102-42dd79c4ddbc">